pilot/pkg/bootstrap/server.go

```diff
@@ -781,7 +781,7 @@ func (s *Server) initEventHandlers() error {
 			}
 			s.EnvoyXdsServer.ConfigUpdate(pushReq)
 		}
-		for _, schema := range collections.Pilot.All() {
+		for _, schema := range collections.PilotServiceApi.All() {
```

Comment:
guard with `if features.EnableServiceApis`, consistent with that in `makeKubeConfigController`

Reply:
Ill do that but this part doesn't really matter, it will ignore CRDs we are not watching in the controller

Reply:
I have changed it